lib/view_component/preview.rb

```diff
@@ -4,7 +4,7 @@
 
 module ViewComponent # :nodoc:
   class Preview
-    include Rails.application.routes.url_helpers if defined?(Rails.application.routes.url_helpers)
+    include Rails.application.routes.url_helpers if Rails && Rails.application.present?
```

Comment:
Do we still need the `defined?` check to support non-Rails environments?

Reply:
Yeah, I think this must've fallen through in a rebate 🧐 I'll make sure to fix it before it gets merged.